### ISSUE TITLE
Consider a MutatingAdmissionWebhook for image prefixes

### ISSUE THREAD

**Issue by moshloop** [synthesis-labs/polaris#20]
e.g.

image-prefix-adder --prefix dcr.{{.AccountId}}.{{.Region}}
image-prefix-adder --prefix harbor.onpremise.local

**Comment by drshade** [comment]
Yes was wanting to use a mutating admission controller - but looked like this required being setup at kubeapi creation - meaning that we can't just drop the operator into any old cluster. The current technique (using a watch on pods and replacing polaris:// ) is totally a hack - but a hack that doesn't have any dependencies.

**Comment by moshloop** [comment]
Looks like kops enables it by default in 1.9 https://github.com/kubernetes/kops/blob/63ba10398144fb004900a1698dd9ea82ade01641/pkg/model/components/apiserver.go#L250
